### ISSUE TITLE
Export code

### ISSUE THREAD

**Issue by mirrortorrent** [alialaa/css-grid-cheat-sheet#3]
would be nice too export the grids that I make, So that I may import to my project and continue to play with the code end.

**Comment by alialaa** [comment]
Yeah I had a suggestion to add an export to codepen button, I think I am gonna work on that when I have some time. Thanks for the suggestion!!

**Comment by Alymosul** [comment]
Would you please add this feature i really need it in my project.

**Comment by alialaa** [comment]
@Alymosul Hello Aly, you can put your project up your arse.
